WireAPI/Sources/WireAPI/Models/Connection/Connection.swift

```diff
@@ -18,7 +18,7 @@
 
 import Foundation
 
-public struct Connection: Equatable {
+public struct Connection: Equatable, Codable {
```

Comment:
I thought that the public objects should only transfer data between layers and not be `Codable`. Adding or modifying properties can lead to unexpected failure on different sides.

If some layer write those objects to disk, for example, it should use it's own internal type.

Reply:
Yeah makes sense, but I'm not sure we can add automatic codable conformance from an extension on the consumer side. Either we add manual conformance (would be painful) or we add mirror models (also painful) that are Codable. Or perhaps there's another solution I'm not seeing right now.

Reply:
My personal way would be to add mirror models and be explicit. But I understand that the simpler way is fine in many cases. It's not a blocker as long as the decision is taken conciously :)

Reply:
I'm leaning toward mirror models too, but I think I'll address this in a separate PR in order to avoid bloating this one even more.

Reply:
I create a ticket for this: https://wearezeta.atlassian.net/browse/WPB-9989